hack/update-deps.sh

```diff
@@ -12,28 +12,40 @@ set -o pipefail
 
 cd ${ROOT_DIR}
 
+# This controls the release branch we track.
+VERSION="master"
+
+# We need these flags for things to work properly.
+export GO111MODULE=on
+export GOFLAGS=-mod=vendor
```

Comment:
Hmm, we aren't running the license code, so we don't need this yet.  Having it here will break `--upgrade`